scriptmodules/supplementary/ps3controller.sh

```diff
@@ -47,27 +47,19 @@ function install_ps3controller() {
 
     cd sixad
     checkinstall -y --fstrans=no
-    insserv sixad
 
     echo "$branch" >"$md_inst/type.txt"
 
     # Disable timeouts
     iniConfig " = " "" "/etc/bluetooth/main.conf"
     iniSet "DiscoverableTimeout" "0"
     iniSet "PairableTimeout" "0"
-
-    # Start sixad daemon
-    /etc/init.d/sixad start
 }
 
 function remove_ps3controller() {
-    /etc/init.d/sixad stop
-    insserv -r sixad
     dpkg --purge sixad
     rm -f /etc/udev/rules.d/99-sixpair.rules
     rm -f /etc/udev/rules.d/10-local.rules
```

Comment:
Not sure if we need to delete these udev rules? The sixad package doesn't provide them...

Reply:
Probably can now - looks like something that was needed from a couple of years ago.